### ISSUE TITLE
Nuget package manager does not highlight selected package when it has no focus

### ISSUE THREAD

**Issue by vsfeedback** [NuGet/Home#12006]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Nuget-package-manager-does-not-highlight/10110029)._

---
In the "Manage NuGet packages" window there is a list of packages (three sub-tabs with those, but it behaves the same in all of them). When you select an item there it is higlighted and details are displayed on the right, but as soon as the window loses focus, that highlight in the list is removed. This is inconsistent with every other list in VS (see solution explorer and find results, they both don't have focus but still show the selected element).

Repro steps:
1. Open NuGet package manager window
2. Select on any package in the list
3. Notice it is highlighted.
4. Press the tab key.

Notice the highlight disappears.

Alternatively, instead of just pressing tab in step 4, click an item in solution explorer to get the focus out of the package manager tab (folder preferably).

Also, getting the focus back into the package manager does not restore the highlight unless you tab through to the list explicitly.

---
#### Feedback Bot on 8/2/2022, 11:24 PM: 

(private comment, text removed)

---
### Original Solutions
(no solutions)

**Comment by heng-liu** [comment]
Moved it to GitHub per @donnie-msft 's suggestion.
It's a similar issue with https://github.com/NuGet/Home/issues/7882
